### PR TITLE
fix: Recovery timestamp was not sending UPF starting time, and node association entity seen time in hearbeats.

### PR DESCRIPTION
**Pull request by marknefedov**

### CODE REVIEW

File: cmd/core/pfcp_hearbeat.go [hunk 1]
```diff
@@ -22,7 +21,7 @@ func HandlePfcpHeartbeatRequest(conn *PfcpConnection, msg message.Message, addr
 		log.Printf("Got Heartbeat Request with TS: %s, from: %s", ts, addr)
 	}
 
-	hbres := message.NewHeartbeatResponse(hbreq.SequenceNumber, ie.NewRecoveryTimeStamp(time.Now()))
+	hbres := message.NewHeartbeatResponse(hbreq.SequenceNumber, ie.NewRecoveryTimeStamp(conn.RecoveryTimestamp))
```
Comment (pirog-spb) [inline cmd/core/pfcp_hearbeat.go:24]: I suppose here we can use association start timestamp as well.

File: cmd/core/pfcp_connection.go [hunk 2]
```diff
@@ -12,13 +12,14 @@ import (
...
-	n3Address        net.IP
-	mapOperations    ebpf.ForwardingPlaneController
+	udpConn           *net.UDPConn
+	pfcpHandlerMap    PfcpHandlerMap
+	NodeAssociations  map[string]*NodeAssociation
+	nodeId            string
+	nodeAddrV4        net.IP
+	n3Address         net.IP
+	mapOperations     ebpf.ForwardingPlaneController
+	RecoveryTimestamp time.Time
```
Comment (pirog-spb) [inline cmd/core/pfcp_connection.go:22]: Don't need it. See comments below

File: cmd/core/node_association.go [hunk 3]
```diff
@@ -15,6 +15,7 @@ type NodeAssociation struct {
 	Sessions         map[uint64]*Session
 	HeartbeatRetries uint32
 	cancelRetries    context.CancelFunc
+	Timestamp        time.Time
```
Comment (pirog-spb) [inline cmd/core/node_association.go:18]: Would be nice to name it properly. `recoveryTimestamp` ? `associationStart`?

File: cmd/core/pfcp_hearbeat.go (hunk 1 above)
Comment (marknefedov) [reply cmd/core/pfcp_hearbeat.go:24]: https://www.etsi.org/deliver/etsi_ts/129200_129299/129244/16.05.00_60/ts_129244v160500p.pdf
6.2.2.2
A CP function or UP function shall be prepared to receive a Heartbeat Request at any time (even from unknown peers) and it shall reply with a Heartbeat Response.

Comment (pirog-spb) [reply cmd/core/pfcp_hearbeat.go:23]: Ок. You convinced me. But make timestamp appopriate name pls. `startTime` or smthg like this.

**Review by pirog-spb — APPROVED** [review]

**Pull request merged**